README.md

```diff
@@ -0,0 +1,2 @@
+# Buildables Fellowship Projects
+Folder structure: task_1/ with .sql and .txt.
```

Comment:
I went through your submission Zaid, good job. A suggestion for you is that on numerous ocassions you have used SubQueries while at that part, thing could've been handled using a simple query as well. Same goes with CTEs, when you can achieve with level 1 CTE no need to go beyond level 1.